src/utils/init.js

```diff
@@ -152,7 +153,9 @@ function setValues(userVals) {
   Object.entries(userVals).forEach(([key, value]) => {
     if (Array.isArray(value))
       result[key] = value.map((entry) => _parseUserVal(entry));
-    else result[key] = _parseUserVal(value);
+    else if (isBase64PNG(value)) {
+      result[key] = _parseUserVal(dataURLToFile(value, `${key}.png`));
```

Comment:
this logic should actually go in _parseUserVal, the array here is to handle repeated fields not an actual field type